api_v3/lib/types/entry/KalturaLiveStreamEntry.php

```diff
@@ -112,6 +112,13 @@ class KalturaLiveStreamEntry extends KalturaLiveEntry
 	 * @readonly
 	 */
 	public $primaryServerNodeId;
+
+	/**
+	 * The low latency flag
+	 *
+	 * @var KalturaNullableBoolean
+	 */
+	public $lowLatency;
```

Comment:
Should everyone be able to enable low latency support or will this require enabling a feature on the partner?